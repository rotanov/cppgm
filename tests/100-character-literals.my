literal 'f' char 66
invalid 'foo'
identifier r
invalid 'bar'
invalid u'bar'
