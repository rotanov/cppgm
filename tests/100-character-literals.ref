literal 'f' char 66
invalid 'foo'
identifier r
invalid 'bar'
invalid u'bar'
invalid U'bar'
invalid L'baz'
eof
