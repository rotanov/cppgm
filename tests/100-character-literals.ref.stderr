ERROR: multi code point character literals not supported: 'foo'
ERROR: multi code point character literals not supported: 'bar'
ERROR: multi code point character literals not supported: u'bar'
ERROR: multi code point character literals not supported: U'bar'
ERROR: multi code point character literals not supported: L'baz'
